workspace/assets/js.json

```diff
@@ -28,12 +27,14 @@
 		"modules/page-load-errors.js",
 		"modules/page-load.js",
 		"modules/change-state.js",
-		"modules/auto-jit-image.js",
```

Comment:
If we remove this, we can also remove both files in lib ("lib/jquery.sizing.min.js", "lib/jquery.jit-image.min.js")

Reply:
Done!